src/vsr/client.zig

```diff
@@ -66,7 +66,7 @@ pub fn Client(comptime StateMachine_: type, comptime MessageBus: type) type {
 
         /// The highest view number seen by the client in messages exchanged with the cluster.
         /// Used to locate the current primary, and provide more information to a partitioned primary.
-        view: u32 = 0,
+        view: u32 = 1,
```

Comment:
Let's motivate why we start with 1 rather than 0, something like: "we count views from 1 to distinguish cases where the view is not set"

Reply:
Ah, it's client.zig again :facepalm: I think we should motivate in `root_prepare`, and here just refer to that.

Reply:
I'm switching back to `view=0` at joran's request, so it's moot.